test/k8sT/Services.go

```diff
@@ -412,6 +412,93 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	Context("Checks local redirect policy", func() {
+		var (
+			deploymentYAML string
+			lrpServiceName string
+			app1IP         string
+			app2IP         string
+			be1IP          string
+			be2IP          string
+		)
+
+		BeforeAll(func() {
+			deploymentYAML = helpers.ManifestGet(kubectl.BasePath(), "lrp-test.yaml")
+			lrpServiceName = "lrp-demo-service"
```

Comment:
Nit: I think this can be declared as a const before `BeforeAll(..)`, can't it?

Reply:
Done.